File: BaekJoon_실습1Level.cpp
#include <iostream>

// 10039번
int main() {
	int n, sum = 0;
	for (int i = 0; i < 5; i++) {
		std::cin >> n;
		if (n < 40)
			n = 40;
		sum += n;
	}
	std::cout << sum / 5;
}

// 5543번
int main() {
	int a[5];
	for (int i = 0; i < 5; i++)
		std::cin >> a[i];
	int min = a[0] + a[3] - 50;
	for (int i = 0; i < 3; i++) {
		for (int j = 3; j < 5; j++) {
			if (min > (a[i] + a[j] - 50))
				min = a[i] + a[j] - 50;
		}
	}
	std::cout << min;
}

//10817번
int main() {
	int a, b, c, mid = 0;
	std::cin >> a >> b >> c;
	if (a > b) {
		if (a > c) {
			if (b > c)
				mid = b;
			else
				mid = c;
		}
		else
			mid = a;
	}
	else {
		if (b > c) {
			if (a > c)
				mid = a;
			else
				mid = c;
		}
		else
			mid = b;
	}
	std::cout << mid;
}

//2523번
int main() {
	int n;
	std::cin >> n;
	for (int i = 0; i < n; i++) {
		for (int j = -1; j < i; j++)
			std::cout << '*';
		std::cout << '\n';
	}
	for (int i = n; i > 0; i--) {
		for (int j = i - 1; j > 0; j--)
			std::cout << '*';
		std::cout << '\n';
	}
}

//2446번
int main() {
	int n, blank = 1;
	std::cin >> n;
	for (int i = 0; i < n; i++) {
		for (int j = 0; j < i; j++)
			std::cout << ' ';
		for (int j = 0; j < 2 * (n-i) - 1; j++)
			std::cout << '*';
		std::cout << '\n';
	}
	for (int i = 1; i < n; i++) {
		for (int j = 1; j < n-i; j++)
			std::cout << ' ';
		for (int j = 0; j < 2 * i + 1; j++)
			std::cout << '*';
		std::cout << '\n';
	}
}

//10996번
int main() {
	int n, even, odd;
	std::cin >> n;
	
	even = n / 2;
	odd = n - n / 2;

	for (int i = 0; i < n; i++) {
		for (int j = 0; j < odd; j++)
			std::cout << "* ";
		std::cout << '\n';
		for (int j = 0; j < even; j++)
			std::cout << " *";
		std::cout << '\n';
	}
}

File: BaekJoon_실습1Level.py
import sys
# # 10039번
# sum = 0
# for i in range(5):
#     n = int(input())
#     if n < 40:
#         n = 40
#     else:
#         n = n
#     sum += n
# print(sum//5)

# # 5543번
# n = []
# for i in range(5):
#     n.append(int(input()))
# min = n[0] + n[3] - 50
#
# for i in range(3):
#     for j in range(3, 5):
#         if min > (n[i] + n[j] - 50):
#             min = (n[i] + n[j] - 50)
# print(min)

# # 10817번
# a, b, c = map(int, map(int, sys.stdin.readline().split()))
#
# if a > b:
#     if a > c:
#         if b > c:
#             mid = b
#         else:
#             mid = c
# elif b > c:
#     if a > c:
#         mid = a
#     else:
#         mid = c
# else:
#     mid = b
#
# print(mid)
#
# # 위의 코드가 정상적으로 작동되지만 계속 런타임 오류가 떠서 아래 코드로 변경하였다.
# a = map(int, input().split())
# sort_a = sorted(a)
# print(sort_a[1])

# # 2523번
# n = int(input())
# for i in range(1, n+1):
#     print('*' * i)
# for i in range(n-1, 0, -1):
#     print('*' * i)

# 2446번
# n = int(input())
# blank = 0
# for i in range(n, 0, -1):
#     print(' ' * blank + '*' * (2*i-1))
#     blank += 1
# blank -= 2
# for i in range(2, n+1, 1):
#     print(' ' * blank + '*' * (2*i-1))
#     blank -= 1

# 10996번
n = int(input())
even = n//2
odd = n - n//2

for i in range(n):
    print('* ' * odd)
    print(' *' * even)
